lib/user_session_management.rb

```diff
@@ -39,16 +42,17 @@ def sign_in!(user, security_log_data: {})
 
     if @current_user.is_anonymous?
       # Clear the SSO cookie
-      sso_cookies.delete(sso_cookie_name)
+      sso_cookie_jar.delete
+      old_sso_cookie_jar.delete('ox')
 
       security_log :sign_out, security_log_data
     else
       session[:last_admin_activity] = DateTime.now.to_s if @current_user.is_administrator?
 
       # Set the SSO cookie
-      sso_cookies.permanent[sso_cookie_name] = {
-        value: { 'user' => Api::V1::UserRepresenter.new(@current_user).to_hash }
-      }
+      cookie = { value: { 'user' => Api::V1::UserRepresenter.new(@current_user).to_hash } }
```

Comment:
Possible to go ahead and add faculty status here?  Will help other efforts.  I think it is not included in the UserRepresenter by default.

Reply:
I think I have to pass in include_private_data

Reply:
Kind of don’t want other data in there eg all emails. I can look at this later. Would be cool if we could just have fac status. Would let other apps enable things without callback to accounts. Maybe faculty status isn’t really private?

Reply:
Kind of don’t want other data in there eg all emails. I can look at this later. Would be cool if we could just have fac status. Would let other apps enable things without callback to accounts. Maybe faculty status isn’t really private?

Reply:
Yeah maybe faculty status is not private (being faculty somewhere is pretty much public knowledge anyway).